resources/views/herd/show.blade.php

```diff
@@ -4,17 +4,35 @@
     <div class="container">
         <div class="row justify-content-center">
             <div class="col-md-8">
-                <div class="text-center mb-4">
-                    <h1><span class="text-muted">{{ $user->name }}</span></h1>
-                    <p class="lead">
-                        @if($user->twitter)
-                            <a href="https://twitter.com/{{ $user->twitter }}">{{ '@' . $user->twitter }}</a> -
-                        @endif
-                        {{ ($country = $countries->get($user->country_code))->get('name') }}
-                        @if ($flag = $country->get('flag'))
-                            <span class="ml-1">{!! $flag !!}</span>
+                <div class="text-left mb-4">
+                    <div class="media">
+                        @if (! $user->twitter)
```

Comment:
according to this logic, the user will have an avatar only if the user has twitter, correct?
for now we are getting avatars from gravatar.com, otherwise, the gravatar's default image.

what do you think on doing:

1. get avatar from twitter if twitter username's available;
2. get from gravatar if available
3. show the initial letters if no gravatar found

Reply:
just to highlight, this happens in `User@avatar()` method